testsuite/E12.py

```diff
@@ -373,3 +373,18 @@ def example_issue254():
 print(a
 , end=' ')
 #:
+def foo():
+    pass
+    raise 123 + \
+           123
+#:
+class Eggs:
+    pass
+    assert 123456 == \
+            123456
+#:
+def f1():
+    print('foo')
+    with open('/path/to/some/file/you/want/to/read') as file_1, \
+          open('/path/to/some/file/being/written', 'w') as file_2:
+        file_2.write(file_1.read())
```

Comment:
none of these three align properly -- they should produce an error I believe?

Reply:
they do.
not sure I understand the syntax of the test file actually... I added `#: E127:4:...` and looks like tests pass
(the error is produced when manually testing as well)

Reply:
:confused: weird!  I'll look into why that is and fix that up :+1: